api/lib/handlers/login/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var config = require('../../config.js').config;
+var pg = require('pg');
+
+module.exports.post = function(req, res, next) {
+  var id = req.body.id;
+  var pool = new pg.Pool(config);
+  pool.connect(function(err, client, done) {
+    if(err) {
+      return console.error('error fetching client from pool', err);
+    }
+    client.query('SELECT * FROM give_me_time_public.person WHERE id=($1)', [id], function(err, result) {
+      done();
+      if(err) {
+        return console.error('error running query', err);
+      }
+      result.rows[0] ? result.rows[0].credit = Math.round(parseInt(result.rows[0].credit)) : '';
```

Comment:
There is a lot of boilerplate involved in doing a single SQL query (2 require, new Pg pool instance, manual error handling, ...)
We could do better with a single service class.
That way you could add support for single row queries and avoid doing `[0]`
It's up to you to decide if you want to use an ORM here.